src/features/game/expansion/components/lavaPit/LavaPitModalContent.tsx

```diff
@@ -145,9 +150,7 @@ export const LavaPitModalContent: React.FC<Props> = ({ onClose, id }) => {
           )}
           {!lavaPitInProgress && (
             <Label icon={SUNNYSIDE.icons.stopwatch} type="transparent">
-              {secondsToString(LAVA_PIT_MS / 1000, {
-                length: "short",
-              })}
+              {secondsToString(lavaPitTime / 1000, { length: "short" })}
```

Comment:
This appears to be rounded down. So instead of 1.5 days, it shows 1 day when you have the necklace on.
<img width="478" alt="Screenshot 2025-05-07 at 11 24 23 AM" src="https://github.com/user-attachments/assets/01b430fb-a709-4641-920f-59ffcfe2a68c" />